Automattic/jetpack
# Super Cache: add an admin notice to encourage migration to Jetpack Boost

For site owners who are using the default settings of WP Super Cache they could use Jetpack Boost and see much the same caching results. This PR adds an admin notice with a "Migrate now" button that installs and activates Boost. It will set a flag that tells Boost to activate the Cache module.

TODO - check the WPSC config matches what Boost supports.

Fixes #37743

## Proposed changes:

* Add a function wpsc_jetpack_boost_notice() that displays the notice. It will only display on the WPSC settings page.
* Add a function wpsc_dismiss_boost_notice() that allows the notice to be hidden.
* Add JavaScript that catches clicks on the dismiss button so that action is recorded by wpsc_dismiss_boost_notice()
* Style the admin notice
* Add a nonce for dismissing the admin notice.

### Other information:


## Jetpack product discussion


pc9hqz-2QH-p2

## Does this pull request change what data or activity we track or use?



## Testing instructions:





* This can't be tested without #37824 and #37797 being merged first.
* To test the admin notice displays, so the dismiss action can be removed, copy wpsc_get_boost_migration_config from #37824
